api/data/seattle_db.go

```diff
@@ -11,16 +11,31 @@ import (
 
 // SeattleOfficer is the object model for SPD officers
 type SeattleOfficer struct {
-	Date            time.Time    `json:"date,omitempty"`
-	Badge           string       `json:"badge,omitempty"`
-	FullName        string       `json:"full_name,omitempty"`
-	Title           string       `json:"title,omitempty"`
-	Unit            string       `json:"unit,omitempty"`
-	UnitDescription nulls.String `json:"unit_description,omitempty"`
-	FirstName       string       `json:"first_name,omitempty"`
-	MiddleName      nulls.String `json:"middle_name,omitempty"`
-	LastName        string       `json:"last_name,omitempty"`
-	Current         bool         `json:"is_current"`
+	Date            string `json:"date,omitempty"`
+	Badge           string `json:"badge,omitempty"`
+	FullName        string `json:"full_name,omitempty"`
+	Title           string `json:"title,omitempty"`
+	Unit            string `json:"unit,omitempty"`
+	UnitDescription string `json:"unit_description,omitempty"`
+	FirstName       string `json:"first_name,omitempty"`
+	MiddleName      string `json:"middle_name,omitempty"`
+	LastName        string `json:"last_name,omitempty"`
+	Current         bool   `json:"is_current"`
+}
+
+// seattleOfficer is an internal intermediary between the returned SQL rows data
+// and the actual JSON return itself.
+type seattleOfficer struct {
```

Comment:
In python it's a convention to precede internal items with an underscore (e.g. `_seattleOfficer`) as a way of denoting for importers which items should and shouldn't be used. Does that make sense to do here, since the names are really similar?

Reply:
So convention wise it's not really used afaik. The leading capital letters tells the compiler that a variable is exported and thus available from the broader scope where as the lowercase (or if we want underscore) denotes that the variable is only available within the scope it is defined in. From what I've seen the lowercase is the variable is effectively the same convention in Go as the underscore is in Python. It's not a big deal either way, we can change it, but just trying to give as much context as I can for why I named it this way.

Reply:
Gotcha! Okay I'm glad there's some convention 👍 that works great